antispam_bee.php

```diff
@@ -1805,50 +1789,7 @@ private static function _is_country_spam($ip)
 		// Dive into whitelist
 		return ( ! in_array($country, $white) );
 	}
-
-	/**
-	* Check for DNSBL spam
-	*
-	* @since   2.4.5
-	* @change  2.4.5
-	*
-	* @param   string   $ip  IP address
-	* @return  boolean       TRUE for reported IP
-	*/
-
-	private static function _is_dnsbl_spam($ip)
-	{
-		// Start request
-		$response = wp_safe_remote_request(
-			esc_url_raw(
-				sprintf(
-					'https://www.stopforumspam.com/api?ip=%s&f=json',
-					$ip
-				),
-				'https'
-			)
-		);
-
-		// Response error?
-		if ( is_wp_error($response) ) {
-			return false;
-		}
-
-		// Get JSON
-		$json = wp_remote_retrieve_body($response);
-
-		// Decode JSON
-		$result = json_decode($json);
-
-		// Empty data
-		if ( empty($result->success) ) {
-			return false;
-		}
-
-		// Return status
-		return (bool) $result->ip->appears;
-	}
-
+	
```

Comment:
Nitpicking: L1792, there is a tab or something.

Reply:
good catch.... but why?! 😱 🤔